### PR TITLE
Adds getBulkSecret.

### PR DESCRIPTION
**Pull request by artursouza**
# Description

Adds getBulkSecret.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #179 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [X] Code compiles correctly
* [X] Created/updated tests
* [X] Extended the documentation

### CODE REVIEW

**Review by wcs1only — DISMISSED** [review]
LGTM

File: dapr/clients/grpc/client.py [hunk 1]
```diff
@@ -700,6 +701,63 @@ def get_secret(
...
             headers=call.initial_metadata())
 
+    def get_bulk_secret(
+            self,
+            store_name: str,
+            secret_metadata: Optional[Dict[str, str]] = {},
+            metadata: Optional[MetadataTuple] = ()) -> GetSecretResponse:
+        """Get secret with a given key.
+
+        This gets a secret from secret store with a given key and secret store name.
```
Comment (mukundansundar) [inline dapr/clients/grpc/client.py:711]: Docstring needs to change

**Review by mukundansundar — CHANGES_REQUESTED** [review]
mostly looks good. Docstring needs update. And typing check to be fixed.

File: dapr/clients/grpc/client.py (hunk 1 above)
Comment (artursouza) [reply dapr/clients/grpc/client.py]: Thanks. Done.

**Review by wcs1only — APPROVED** [review]
Thanks for adding README.md validation for secret store example!

**Review by mukundansundar — APPROVED** [review]

**Pull request merged**